exercises/ledger/canonical-data.json

```diff
@@ -0,0 +1,240 @@
+        "locale": "en_US",
+        "entries": [
+          {
+            "date": "2015-01-02",
+            "description": "Get present",
+            "amount": 1000
+          },
+          {
+            "date": "2015-01-01",
+            "description": "Buy present",
+            "amount": -1000
+          }
+        ]
+      },
+      "expected": [
+        "Date       | Description               | Change       ",
+        "01/01/2015 | Buy present               |      ($10.00)",
+        "01/02/2015 | Get present               |       $10.00 "
+      ]
+    },
+    {
+      "uuid": "29dd3659-6c2d-4380-94a8-6d96086e28e1",
+      "description": "final order tie breaker is change",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "en_US",
+        "entries": [
+          {
+            "date": "2015-01-01",
+            "description": "Something",
+            "amount": 0
+          },
+          {
+            "date": "2015-01-01",
+            "description": "Something",
+            "amount": -1
+          },
+          {
+            "date": "2015-01-01",
+            "description": "Something",
+            "amount": 1
+          }
+        ]
+      },
+      "expected": [
+        "Date       | Description               | Change       ",
+        "01/01/2015 | Something                 |       ($0.01)",
+        "01/01/2015 | Something                 |        $0.00 ",
+        "01/01/2015 | Something                 |        $0.01 "
+      ]
+    },
+    {
+      "uuid": "9b9712a6-f779-4f5c-a759-af65615fcbb9",
+      "description": "overlong description",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "en_US",
+        "entries": [
+          {
+            "date": "2015-01-01",
+            "description": "Freude schoner Gotterfunken",
+            "amount": -123456
+          }
+        ]
+      },
+      "expected": [
+        "Date       | Description               | Change       ",
+        "01/01/2015 | Freude schoner Gotterf... |   ($1,234.56)"
+      ]
+    },
+    {
+      "uuid": "67318aad-af53-4f3d-aa19-1293b4d4c924",
+      "description": "euros",
+      "property": "formatEntries",
+      "input": {
+        "currency": "EUR",
+        "locale": "en_US",
+        "entries": [
+          {
+            "date": "2015-01-01",
+            "description": "Buy present",
+            "amount": -1000
+          }
+        ]
+      },
+      "expected": [
+        "Date       | Description               | Change       ",
+        "01/01/2015 | Buy present               |      (€10.00)"
+      ]
+    },
+    {
+      "uuid": "bdc499b6-51f5-4117-95f2-43cb6737208e",
+      "description": "dutch locale",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "nl_NL",
+        "entries": [
+          {
+            "date": "2015-03-12",
+            "description": "Buy present",
```

Comment:
I'm a bit reluctant to change this when we keep everything else as English. The formatting is supposed to use `nl_NL` but the headers are still English.

Reply:
I think this might make the tests every so slightly more colorful. I also fail to see serious downsides. Am I missing any?

If easter eggs are welcome in cases such as this, I propose `"Nieuwe flessenlikker"` (or some such) instead of `"Koop cadeau"`.

Reply:
I'm a _bit reluctant_, not very strongly opposed. I'm leaning towards keeping exercises uniformly English, as Exercism tries to keep everything in English. I don't see any "serious downsides" ... but I don't see any downside to keeping it as is, either 😄

Reply:
Okay, let's keep it in English